openedx/core/djangoapps/programs/tests/test_backpopulate_program_credentials.py

```diff
@@ -81,42 +79,38 @@ def test_handle(self, commit, mock_task, mock_get_programs):
     @ddt.data(
         [
             factories.Program(
-                organizations=[factories.Organization()],
-                course_codes=[
-                    factories.CourseCode(run_modes=[
-                        factories.RunMode(course_key=course_id),
+                courses=[
+                    factories.Course(course_runs=[
+                        factories.CourseRun(key=course_id),
```

Comment:
That is technically more correct, yes. Traditionally, edx-platform has used `course_id` to refer to a course run ID, but I'll make the name more explicit here.